src/app.coffee

```diff
@@ -32,6 +31,8 @@ knex.init.then ->
 
 		bootstrap.done
 		.then ->
+			# The device module communicates with the api and as such needs bootstrapping to complete
+			device = require './device'
```

Comment:
It seems like it should be ok to do now, but I remember it causing issues before I moved it - it's likely something changed in the meantime